test/supports.js

```diff
@@ -78,14 +78,14 @@ describe('should support for each CSS in JS package with .babelrc', () => {
 	beforeAll(() => {
 		fs.writeFileSync(
 			'.babelrc',
-			`{		
+			`{
 			"plugins": [["@babel/plugin-proposal-decorators", { "legacy": true }]]
 		}`,
 		);
 	});
 
 	afterAll(() => {
-		fs.rmSync('.babelrc');
+		fs.unlinkSync('.babelrc');
```

Comment:
[note] `fs.rmSync` has been added in v14.14.0. I don't know why it's been working on Node.12 until now. 🤷🏼 

See also:
- [`fs.rmSync`](https://nodejs.org/api/fs.html#fsrmsyncpath-options)
- [`fs.unlinkSync`](https://nodejs.org/api/fs.html#fsunlinksyncpath)